pypeit/par/pypeitpar.py

```diff
@@ -1153,6 +1154,11 @@ def __init__(self, offsets=None, weights=None, use_slits4wvgrid=None):
         dtypes['weights'] = [str, list]
         descr['weights'] = 'Mode for the weights used to coadd images.  See coadd2d.py for all options.'
 
+        # Weights
+        defaults['manual'] = ''
+        dtypes['manual'] = str
+        descr['manual'] = 'Manual extraction parameters. det:spat:spec:fwhm. comma separated '
```

Comment:
You might add a note that these are the spat:spec positions in the pseudo-image produced by the 2d coadd.